django/contrib/postgres/fields/ranges.py

```diff
@@ -132,6 +133,44 @@ def db_type(self, connection):
 RangeField.register_lookup(lookups.Overlap)
 
 
+class DateTimeRangeContains(models.Lookup):
```

Comment:
It might be worth adding a docstring mentioning that the sole purpose of this lookup subclass is to work around a limitation of Psycopg2/PostgreSQL in regard to type casting?